components/TitleSection.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+const TitleSection = styled.section`
+  border-bottom-style: solid;
+  border-bottom-color: var(--primary-color);
+  border-bottom-width: 2px;
```

Comment:
```suggestion
  border: 2px solid var(--primary-color);
```

Reply:
Changed!

Reply:
I had to change it back, because the solution you suggested didn't work on me page (even with border-bottom:...).